README.md

```diff
@@ -37,6 +37,12 @@ You find an example in [`test/template-escape`](test/template-escape).
 If you want to use for example `$(_name)` as both an external reference and a normal reference,
 then you add a `\` for the latter resulting in `$(\_name)` for the latter.
 
+If your YARRRML document contains sensitive information such as database credentials,
+you can use dollar variables with curly braces and specify a variable in a `.env` file.
+For example, use `${DB_PASSWORD}` in the YARRRML document and add a `.env` file with content `DB_PASSWORD=mySecretPassword`.
```

Comment:
Why can't we use normal brackets as well here? And check all variables if they are defined in the `.env`?

Reply:
Or do the same as with the external references? Adding a `_` before the variable name? so you would have then `$(_DB_PASSWORD_)`? So external references can come from both the CLI and `.env`, which makes sense.

Reply:
You only have to change your YARRRML files, nothing else. The variables names remain the same. You only add the `_` in the YARRRML template.